wellcomeml/ml/clustering.py

```diff
@@ -2,19 +2,27 @@
 import logging
 import os
 
+from wellcomeml.ml import vectorizer
+from wellcomeml.logger import logger
+
 import numpy as np
 from sklearn.base import ClusterMixin
 from sklearn.cluster import DBSCAN, OPTICS, KMeans
 from sklearn.manifold import TSNE
 from sklearn.model_selection import GridSearchCV
 from sklearn.metrics import silhouette_score
 from sklearn.pipeline import Pipeline
-#  from hdbscan import HDBSCAN
+try:
+    from hdbscan import HDBSCAN
+    HDBSCAN_INSTALLED = True
+except ValueError:
```

Comment:
I'm really surprised this is not a `ImportError`

Reply:
You can try it out by commenting out the flags in the install
`$(VIRTUALENV)/bin/pip3 install hdbscan #--no-cache-dir --no-binary :all: --no-build-isolation`

Any way we can add a test for this?